package.json

```diff
@@ -201,7 +201,7 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "^1.0.0",
+    "@ciscospark/eslint-config": "^2.0.0",
```

Comment:
I suggested they publish the new package as a 2.0 to completely separate it from the js-sdk package.